huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```
@@ -0,0 +1,530 @@
+	}
+	return bodyParams
+}
+
+func cceAccessConfigResourceImportState(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	var (
+		cfg                        = meta.(*config.Config)
+		region                     = cfg.GetRegion(d)
+		listCceAccessConfigProduct = "lts"
+		name                       = d.Id()
+	)
+	d.Set("name", name)
+	ltsClient, err := cfg.NewServiceClient(listCceAccessConfigProduct, region)
+	if err != nil {
+		return []*schema.ResourceData{d}, fmt.Errorf("error creating LTS client: %s", err)
+	}
+
+	return []*schema.ResourceData{d}, refreshCceAccessID(ltsClient, d)
+}
+
+func refreshCceAccessID(ltsClient *golangsdk.ServiceClient, d *schema.ResourceData) error {
+	var (
+		listCceAccessConfigHttpUrl = "v3/{project_id}/lts/access-config-list"
+	)
+	listCceAccessConfigPath := ltsClient.Endpoint + listCceAccessConfigHttpUrl
+	listCceAccessConfigPath = strings.ReplaceAll(listCceAccessConfigPath, "{project_id}", ltsClient.ProjectID)
+
+	listCceAccessConfigOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	name := d.Get("name").(string)
+	listCceAccessConfigOpt.JSONBody = map[string]interface{}{
+		"access_config_name_list": []string{name},
+	}
+	listCceAccessConfigResp, err := ltsClient.Request("POST", listCceAccessConfigPath, &listCceAccessConfigOpt)
+	if err != nil {
+		return fmt.Errorf("error retrieving CCE access config: %s", err)
+	}
+
+	listCceAccessConfigRespBody, err := utils.FlattenResponse(listCceAccessConfigResp)
+	if err != nil {
+		return err
+	}
+
+	jsonPath := fmt.Sprintf("result[?access_config_name=='%s']|[0]", name)
+	listCceAccessConfigRespBody = utils.PathSearch(jsonPath, listCceAccessConfigRespBody, nil)
+	if listCceAccessConfigRespBody == nil {
+		return fmt.Errorf("the CCE access config (%s) does not exist", name)
+	}
+	configID := utils.PathSearch("access_config_id", listCceAccessConfigRespBody, "")
+	if configID != "" {
+		d.SetId(configID.(string))
+	}
+	return nil
```

shall we return any error message if the access config ID is not exist?

---

done